ee/clickhouse/queries/test/__snapshots__/test_trends.ambr

```diff
@@ -2045,7 +2045,7 @@
     (SELECT any(session_duration) as session_duration,
             breakdown_value
      FROM
-       (SELECT $session_id,
+       (SELECT sessions."$session_id",
```

Comment:
@neilkakkar This is kind of a head scratcher. If we label this table as "e" it works when there's a person join but doesn't work when there's not. (Take a look at the failing test on the first commit)